lib/clients/jira.go

```diff
@@ -361,7 +418,26 @@ func (j dryrunJIRAClient) ListIssues(ids string) ([]jira.Issue, error) {
 		return nil, fmt.Errorf("Get JIRA issues failed: expected []jira.Issue; got %T", ji)
```

Comment:
Do not capitalize this error either. https://github.com/golang/go/wiki/CodeReviewComments#error-strings